recipes/configs/phi3/mini_full.yaml

```diff
@@ -41,21 +41,20 @@ checkpointer:
     model-00002-of-00002.safetensors
   ]
   recipe_checkpoint: null
-  output_dir: /tmp/Phi-3-mini-4k-instruct
+  output_dir: /tmp/finetuned-phi3-4k-model
```

Comment:
What's with the output dir? Personally I'd just keep it in the same location for now for consistency with existing configs then we can tackle the changes we discussed around that across the board in a separate PR

Reply:
a;lskdjf;laksdjf;laksdjf;lkasdjf;lkasjd;flkasjd;flkasjd;flkasjd;lfkasjd;flkasjd;lf

Reply:
But I hate it so much